### ISSUE TITLE
maybe a bug?

### ISSUE THREAD

**Issue by JSchmidt63** [energye/energy#14]
1. first create a clean energy by downloaded [energy_cmd_windows64](https://gitee.com/energye/assets/releases/download/v1.0.0/energy_cmd_windows64.zip)
2. windows -> cmd/energy.exe install . finally:  Automatically configure the CEF and Energy framework SUCCESS Version: latest
3. go mod init simple
4. go mod tidy
5. create the simple.go from examples/simple
6. go run simple.go
7. i get some error
```
PS C:\Users\js\Desktop\AntD\a_broiler\project\energy2> go run simple.go  
The specified procedure could not be found.
The specified procedure could not be found.
The specified procedure could not be found.
The specified procedure could not be found.
The specified procedure could not be found.
The specified procedure could not be found.
The specified procedure could not be found.
The specified procedure could not be found.
The specified procedure could not be found.
The specified procedure could not be found.
The specified procedure could not be found.
PS C:\Users\js\Desktop\AntD\a_broiler\project\energy2> 
```
Cheers

```
package main

import (
	"github.com/energye/energy/v2/cef"
)

func main() {
	//Global initialization must be called by every application
	cef.GlobalInit(nil, nil)
	//Create application
	cefApp := cef.NewApplication()
	//Set URL
	cef.BrowserWindow.Config.Url = "https://energy.yanghy.cn"
	//Run App
	cef.Run(cefApp)
}
```

```
module simple

go 1.20

require github.com/energye/energy/v2 v2.0.2

require (
	github.com/energye/golcl v0.0.0-20230607072628-c317d3f83490 // indirect
	github.com/godbus/dbus/v5 v5.1.0 // indirect
	github.com/json-iterator/go v1.1.12 // indirect
	github.com/modern-go/concurrent v0.0.0-20180228061459-e0a39a4cb421 // indirect
	github.com/modern-go/reflect2 v1.0.2 // indirect
	github.com/tevino/abool v0.0.0-20220530134649-2bfc934cb23c // indirect
	golang.org/x/sys v0.0.0-20220722155257-8c9f86f7a55f // indirect
)
```

**Comment by sxmxta** [comment]
1. Check if the '' environment dependency is correctly installed (CEF, liblcl)

2. You can try manually downloading the latest version of liblcl. dll from windows64 and replacing it in the 'ENERGY_HOME' environment variable directory

Once the environment is installed correctly, there is no problem

You can refer to the documentation [link](https://energy.yanghy.cn/#/course/6342d92c401bfe4d0cdf6065/63511b14a749ba0318943f3a)

**Comment by JSchmidt63** [comment]
i checked you mentions.
and i try: energy install .

now is it perfect :-)
Cheers

**Comment by sxmxta** [comment]
latest version is complete

If you still have this problem, check the liblcl Dynamic-link library under the environment variable 'ENERGY_HOME'

You can delete this environment variable and reconfigure it, manually configure it, or reinstall it using the command line after deletion

**Comment by sxmxta** [comment]
You should be a previous version that has not been successfully upgraded and updated

**Comment by sxmxta** [comment]
This issue has been fixed in v2.1.0

**Comment by sunvim** [comment]
energy install failed: 

```shell
Automatic installation and configuration of the energy framework complete development environment
Getting the latest version information from remote service...
ERROR Get "https://energy.yanghy.cn/data/latest-version.json": tls: failed to
                      │   verify certificate: x509: certificate is valid for *.gitee.com, gitee.com,
                      │   not energy.yanghy.cn
```
